pkg/network/master/egressip.go

```diff
@@ -36,6 +37,7 @@ type egressIPManager struct {
 
 type egressNode struct {
 	ip      string
+	sdnIP   string
```

Comment:
These changes to `pkg/network/master/egressip.go` don't seem to do anything? Because nothing ever uses `sdnIP`.

Oh... ugh... the master code is pinging the hostnetwork IPs rather than the SDN IPs... That shouldn't work... (The firewalls ought to be blocking access to the port it uses.) Hm... I guess if people only use egress IPs on bare metal, and they aren't bothering to set up fully-restrictive firewalls, then this would work... But it's still wrong.

The code in `check()` that calls `Ping()` should be fixed to use the sdn IP rather than the node IP... but that's a big change and not related to this bug, so let's do that as a second commit, and then _not_ backport that commit right away. (So, the first commit would have no changes to `pkg/network/master/egressip.go` except for updating the signature of `ClaimEgressIP`, and then the second commit would add these changes back in [except without the extra `ParseCIDR` error checks because those should be unneeded here too] and also fix the `Ping()` call.)

Reply:
No, you didn't miss anything; the code was wrong before your PR